client/pages/Event/EventDetailsPage.jsx

```diff
@@ -115,16 +117,33 @@ class EventDetailsPage extends React.Component {
               {this.renderDeleteEventButton()}
             </div>
           ) : (
-            <button
-              type="button"
-              onClick={this.rsvpEvent}
-              className="event-details__rsvp_button"
-              tooltip={message}
-              disabled={ message ? ' disabled' : null}
-            >
-              {' '}
-              Attend &#10004;
-            </button>
+            <Fragment>
+              <button
+                type="button"
+                onClick={this.rsvpEvent}
+                className="event-details__rsvp_button"
+                tooltip={message}
+                disabled={message ? ' disabled' : null}
+              >
+                {' '}
+                Attend &#10004;
+              </button>
+              <button
+                type="button"
+                onClick={this.showSlackChannels}
+                className="event-details__slack_button"
+              >
+                {' '}
+                Share on Slack <i class="fab fa-slack" />
+              </button>
+              {this.state.showSlackChannels && (
+                <div className="event-details__menu">
```

Comment:
Great job @samdiano on this implementation. It is however important to note that since we will be fetching the list of channels from the backend, it would be nice to make this dropdown as dynamic as possible. Instead of hardcoding the values at the markup level, please use fixtures and then map over your fixture list to generate the individual list items. Please reach out to @ipheghe to get an idea of the structure of data returned by the backend. Also. please make sure the dropdown scrolls when it overflows a certain height and test its responsiveness. Cheers.

Reply:
Alright